app/views/batch_contexts/_new_bc_form.erb

```diff
@@ -73,63 +82,56 @@
-            <%= form.input :run_stt, label: 'Would you like to auto-generate caption/transcript files for the media?',
-                input_html: { data: { action: "change->caption#runSttChanged" } },
-                as: :radio_buttons,
-                collection: [['Yes', true], ['No', false]] %>
-            <div data-caption-target="runSttNotes" hidden>
-                <%= render 'alert', alert_type: 'info', text: :stt_accessibility %>
-                <%= render 'alert', alert_type: 'warning', text: :stt_media %>
+        <div data-caption-target="sttSettings" hidden>
+            <div data-caption-target="sttAvailable">
+                <%= form.input :stt_available, label: 'Do you have caption/transcript files for the media?', input_html: { data: { action: "change->caption#sttAvailableChanged" } },
+                    as: :radio_buttons,
+                    collection: [['Yes', true], ['No', false]] %>
+            </div>
+            <div data-caption-target="manuallyCorrectedStt" hidden>
+                <%= render 'alert', alert_type: 'info', text: :stt_corrected %>
+            </div>
+            <div data-caption-target="runStt">
+                <%= form.input :run_stt, label: 'Would you like to auto-generate caption/transcript files for the media?',
+                    input_html: { data: { action: "change->caption#runSttChanged" } },
+                    as: :radio_buttons,
+                    collection: [['Yes', true], ['No', false]] %>
+                <div data-caption-target="runSttNotes" hidden>
+                    <%= render 'alert', alert_type: 'info', text: :stt_accessibility %>
+                    <%= render 'alert', alert_type: 'warning', text: :stt_media %>
+                </div>
             </div>
         </div>
-    </div>
 
-    <%= form.input :staging_location, input_html: { data: { "globus-target": "stagingLocation" } } %>
+        <%= form.input :staging_location, input_html: { data: { "globus-target": "stagingLocation" } } %>
 
-    <%# the 'staging_style_symlink` is hidden, since we do not currently want users to access it (it may cause problems with some accessioning steps)%>
-    <%= form.hidden_field :staging_style_symlink, value: false %>
+        <%# the 'staging_style_symlink` is hidden, since we do not currently want users to access it (it may cause problems with some accessioning steps)%>
+        <%= form.hidden_field :staging_style_symlink, value: false %>
 
-    <%= form.input :using_file_manifest,
-                   as: :radio_buttons,
-                   collection: [['Yes', true], ['No', false]],
-                   label: 'Do you have a file manifest?' %>
+        <div data-caption-target="usingFileManifest">
+            <%= form.input :using_file_manifest,
+                        as: :radio_buttons,
+                        collection: [['Yes', true], ['No', false]],
+                        label: 'Do you have a file manifest?' %>
+        </div>
```

Comment:
the only actual change here is adding a wrapping div to apply the `usingFileManifest` target to we can hide/show it in the JS controller